content_arfs/code/ghost_pods_arfs.dm

```diff
@@ -36,10 +36,8 @@ var/global/list/pokemon_pods = list()//List of pods that ghosts can spawn at
 	if(LAZYLEN(pokemon_choices_list))
 		return FALSE	//The global list is already generated
 	var/pokemon_choices_list_paths = typesof(/mob/living/simple_mob/animal/passive/pokemon) - remove_paths
-	for (var/path in pokemon_choices_list_paths)//add the mobs to a list with their names referencing paths
-		var/mob/living/simple_mob/animal/passive/pokemon/P = new path()
-		pokemon_choices_list["[P.name]"] = P.type
-		del(P)
+	for (var/mob/living/simple_mob/animal/passive/pokemon/path in pokemon_choices_list_paths)//add the mobs to a list with their names referencing paths
+		pokemon_choices_list["[initial(path.name)]"] = path
```

Comment:
Clever!

Reply:
I did make sure it wasn't modified at runtime or anything. The ditto stuff has some special cases to handle a null tt_desc since that's sometimes set at runtime.